app/src/components/widgets/singleLevelWidgets/charts/failedCasesTrendChart/failedCasesTrendChart.jsx

```diff
@@ -64,6 +81,20 @@ export class FailedCasesTrendChart extends Component {
     this.node.addEventListener('mousemove', this.setupCoords);
   };
 
+  onChartClick = (data) => {
+    const { widget, getStatisticsLink } = this.props;
+    const launchesId = this.config.data.launchesId;
+    const link = getStatisticsLink({
+      statuses: [FAILED],
+    });
+    const navigationParams = this.getDefaultParamsWidget(
```

Comment:
`getDefaultParamsWidget` sounds unclear. Please rename it to `getDefaultLinkParams` or `getCommonLinkParams`.